draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -520,6 +528,83 @@ Author:  Paul Hoffman, paul.hoffman@icann.org
 Change controller:  IESG
 ~~~~~
 
+# Privacy Considerations {#PrivacyConsiderations}
+
+{{RFC7626}} discusses DNS Privacy Considerations in both "On the
+wire" (Section 2.4), and "In the server" (Section 2.5) contexts. This
+is also a useful framing for DoH's privacy considerations.
+
+## On The Wire {#OnTheWire}
+
+DoH encrypts DNS traffic and requires authentication of the
+server. This mitigates both passive surveillance {{RFC7258}} and
+active attacks attempting to divert DNS traffic to rogue servers
+({{RFC7626}} Section 2.5.1). DNS over TLS {{RFC7858}} provides
+similar protections, while direct UDP and TCP based transports are
+vulnerable to this class of information leak.
+
+Additionally, the use of the HTTPS default port 443 and the ability to
+mix DoH traffic with other HTTPS traffic on the same connection can
+deter on-path devices from interfering with DNS operations and make
+DNS traffic analysis more difficult.
+
+## In The Server {#InTheServer}
+
+A DoH implementation is built on IP, TCP, TLS, and HTTP. Each layer
```

Comment:
Technically, couldn't DoH be built on IP, UDP, QUIC, and HTTP?

Reply:
not in any existing standards document I could find :)..  but definitely, if quic is an https transport then it works for doh.

I actually thought about this for the text quite a bit, but I think it enumerating it all just complicates the text without really changing anything that's important for the reader to consider.. in quic, udp and ip more or less merge and the TCP and TLS cookies both live in the quic layer roughly the same way (as address validation and session tickets..). Its the nature of quic to make this harder to understand.